templates/site/node.html

```diff
@@ -1,62 +1,66 @@
-                <div class="row">
-                    <div class="col-9">
-                        <!--H3 title-->
-                        <div id="affiliation" class="mb-1 text-body-secondary"></div>
-                        <div id="description"></div>
-                        <div class="mt-3">
-                            <table>
-                                <thead>
-                                <tr>
-                                    <th></th>
-                                </tr>
-                                </thead>
-                                <tbody>
-                                <tr>
-                                    <td>
-                                        <div id="url"></div>
-                                    </td>
-                                </tr>
-                                <tr>
-                                    <td>Date Added: <span id="date_added"></span></td>
-                                </tr>
-                                <tr>
-                                    <td>Status: <span id="status"></span></td>
-                                </tr>
-                                <tr>
-                                    <td>Contact: <span id="contact"></span></td>
-                                </tr>
-                                <tr>
-                                    <td>Version: <span id="version"></span></td>
-                                </tr>
-                                </tbody>
-                            </table>
-                        </div>
-                        <div id="registration_status" class="mt-3"></div>
-                        <div id="registration_link"></div>
-                    </div>
-                    <div class="col-3 col-lg-2 d-block d-lg-block">
-                        <img id="image-right" class="w-sm-100 mt-2 px-auto d-block img-fluid mw-25 mh-25" src="" />
-                    </div>
-                </div>
-                <div class="row">
-                    <div class="col-12 mt-2 text-center">
-                        <h4 class="text-decoration-underline">Services Available</h4>
-                    </div>
-                </div>
-                <div class="row">
-                    <div class="col-12 mb-2">
-                        <div id="services" class="table-responsive border rounded"></div>
-                    </div>
-                </div>
+   {% include "partials/banner.html" %}
+    {% with %}
+        {% set href = href|default("{{href}}") %}
+        {% set section_title = "Start a Project" %}
+        {% set section_content = "Select most suitable node or server to start a project" %}
+
+        {% include "partials/title-and-text.html" %}
+    {% endwith %}
+
+    <div class="container-fluid w-75">
+        <div class="row">
+            <div id="nodeMenu"></div>
+        </div>
+<!--
```

Comment:
remove

Reply:
Commented parts removed, but page still in development.